### PR TITLE
translate get-dogecoin.md into Chinese

### PR DESCRIPTION
**Pull request by shuferhoo**
translate get-dogecoin.md into Chinese

### CODE REVIEW

**Comment by bobdos** [comment]
hi @shuferhoo 

do you have a dogecoin wallet? 

then I can donate some to you to say thank you for your contribution, personally.

**Comment by shuferhoo** [comment]
> hi @shuferhoo
> 
> do you have a dogecoin wallet?
> 
> then I can donate some to you to say thank you for your contribution, personally.



> hi @shuferhoo
> 
> do you have a dogecoin wallet?
> 
> then I can donate some to you to say thank you for your contribution, personally.

thank you, i have a wallet DKZU4AYzvbR8W9MSUrUG8ACsdjMo1U3tkB. I am happy to be a dogecoin contributor.

**Comment by bobdos** [comment]
> 

cool, just donate Ð420 to you:

https://blockchair.com/dogecoin/address/DKZU4AYzvbR8W9MSUrUG8ACsdjMo1U3tkB

cheers,

**Comment by shuferhoo** [comment]
@bobdos i have received it,  thank you so much as it is my first being tipping!
Please merge my pull request，i am very looking forward to it，hahahah

File: content/zh-cn/dogepedia/articles/get-dogecoin.md [hunk 1]
```diff
@@ -1,33 +1,32 @@
...
-### Exchanges 
+有很多方法可以获得狗狗币，比如从交易所购买，比如通过小费打赏，比如提供商品和服务获得狗狗币，还可以通过参与一些项目比如Dogecoin Folding @ Home。
+
+### 交易所
 
-A popular way people get their hands on Dogecoin is via an exchange. There are many exchanges on the market, and they are most commonly used to exchange fiat money for cryptocurrency, or to exchange cryptocurrency for fiat money.  
+获得狗狗币的一种常见方法是从交易所购买。市面上有很多交易所，你可以在交易所用法定货币购买加密货币，也可以用加密货币兑换法定货币。
 
-You can purchase Dogecoin on an exchange at the current market price (this is called a market *taker* transaction), or by making an offer to purchase cryptocurrency once it reaches a specified price (a market *maker* transaction). Some exchanges only allow its users to purchase cryptocurrencies via market taker transactions.  
+你可以在交易所按照当前市场价格购买狗狗币，这被称为现价交易。或者你可以出一个价格，当市场价格到达这个价格的时候就会达成交易，这被称为限价交易。有一些交易所只允许他们的用户进行市价交易。
```
Comment (bobdos) [inline content/zh-cn/dogepedia/articles/get-dogecoin.md:16]: "你可以在交易所按照当前市场价格购买狗狗币，这被称为现价交易" 中 ”现价“调整为”市价“，这样和段落结尾的”市价交易“的翻译保持一致性。

File: content/zh-cn/dogepedia/articles/get-dogecoin.md [hunk 2]
```diff
@@ -1,33 +1,32 @@
...
 
-Although the security of exchanges has come a long way in recent years, it is still crucial that you **do your own research (DYOR)** before purchasing Dogecoin via an exchange, because they are not immune to security risks or controversy. For added security, it is also important to consider keeping the Dogecoin that you do not intend to use for trading in a [private non-custodial wallet](/dogepedia/articles/how-do-i-get-a-wallet).
+尽管现在交易所的安全性已经有了很大提升，但是你在通过交易所购买狗狗币钱还是需要自己进行仔细的研究，一些风险和争议性问题仍然有可能出现。更安全起见，最好还是考虑把你的短期不打算用于交易的狗狗币放到一个[私有非托管钱包](/zh-cn/dogepedia/articles/how-do-i-get-a-wallet)里。
```
Comment (bobdos) [inline content/zh-cn/dogepedia/articles/get-dogecoin.md:18]: ”自己进行仔细的研究“ 需要和原文**do your own research (DYOR)**一样保持粗体状态

File: content/zh-cn/dogepedia/articles/get-dogecoin.md [hunk 3]
```diff
@@ -1,33 +1,32 @@
...
+### 提供商品和服务
 
+你在经营一个小公司吗？ 现在，通过很多方便的第三方支付方案，提供商品和服务以收取狗狗币变得非常容易。[点击这里](/zh-cn/dogepedia/articles/how-can-my-business-accept-dogecoin)以获取更多信息。
 
-### Other 
+### 其他 
 
-There are other ways you can get your hands on Dogecoin, too. Popular initiatives like [Dogecoin Folding @ Home](/dogepedia/articles/dogecoin-folding-at-home), and even [mining Dogecoin](/dogepedia/how-tos/mining-dogecoin), can help people get their hands on some Doge! 
\ No newline at end of file
+这里还有很多方法可以获得狗狗币。一些活动比如[Dogecoin Folding @ Home](/zh-cn/dogepedia/articles/dogecoin-folding-at-home),包括[挖矿](/zh-cn/dogepedia/how-tos/mining-dogecoin)都可以获得一些狗狗币!
```
Comment (bobdos) [inline content/zh-cn/dogepedia/articles/get-dogecoin.md:32]: 本段落中的英文逗号"," 需要 转换为 中文逗号"，"

**Review by bobdos — CHANGES_REQUESTED** [review]

**Comment by shuferhoo** [comment]
@bobdos ok, i have changed all these requests.

**Review by bobdos — APPROVED** [review]
all good and it's ready to go.

many thanks for your contribution.

cheers.

**Pull request merged**